src/core/packet_builder.c

```diff
@@ -205,7 +205,7 @@ QuicPacketBuilderPrepare(
         uint16_t NewDatagramLength =
             MaxUdpPayloadSizeForFamily(
                 QuicAddrGetFamily(&Builder->Path->RemoteAddress),
-                IsPathMtuDiscovery ? CXPLAT_MAX_MTU : DatagramSize);
+                IsPathMtuDiscovery ? Builder->Path->MtuDiscovery.ProbedSize : DatagramSize);
```

Comment:
`ProbedSize` in this context seems to indicate this size has already been probed. That's not true, right? So `ProbeSize` might be better?

Reply:
Its possible this is a repeat probe. Its the current probe size. So yeah ProbeSide might work better.